rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketController.java

```diff
@@ -196,13 +197,13 @@
     if (workbasketId.equals(workbasketResource.getWorkbasketId())) {
       Workbasket workbasket = workbasketResourceAssembler.toModel(workbasketResource);
       workbasket = workbasketService.updateWorkbasket(workbasket);
-      result = ResponseEntity.ok(workbasketResourceAssembler.toResource(workbasket));
+      result = ResponseEntity.ok(workbasketResourceAssembler.toModel(workbasket));
     } else {
       throw new InvalidWorkbasketException(
           "Target-WB-ID('"
               + workbasketId
               + "') is not identical with the WB-ID of to object which should be updated. ID=('"
-              + workbasketResource.getId()
+              + workbasketResource.getLink(IanaLinkRelations.SELF)
```

Comment:
this should be workbasketResource.getWorkbasketId()

Reply:
From my understanding of the previous implementation of the function the getId() method came from the ResouceSupport class and returns a Type of Link with a relation of self. As such replacing it with a method that returns a type of String is not a valid replacement in my eyes.